src/frontend/hooks/useHentBehandling.ts

```diff
@@ -0,0 +1,31 @@
+import { byggTomRessurs, Ressurs } from '../typer/ressurs';
+import { useApp } from '../context/AppContext';
+import { useCallback, useState } from 'react';
+import { Behandling } from '../typer/fagsak';
+import { AxiosRequestConfig } from 'axios';
+
+export const useHentBehandling = (
+    behandlingId?: string
```

Comment:
behandlingId for alle disse useHent er vel required? Og burde finnes allerede fra contexten (useParam) ?

Reply:
ja kan fjerne den her !